src/EditorKeyMap.js

```diff
@@ -203,3 +188,18 @@ export const ALL_KEYS = [
   KEY_TOGGLE_UNDERLINE,
   KEY_UNDO,
 ];
+
+export function findKeymapByDescription(description: string): ?Keymap {
+  const matches = ALL_KEYS.filter(keymap => {
+    return keymap.description.toUpperCase() === description.toUpperCase();
+  });
+  return matches[0];
+}
```

Comment:
Terribly inefficient.  There are 28 entries in the keymap.   Using this method, the same description string is converted to upper-case 28 times even if the very first entry is a match.

```
export function findShortcutByDescription(description: string): ?Keymap{
     // Convert to upper case ONCE for case insensitive compare.
     description = description.toUpperCase();
     // Stop searching and return when the first match has been found.
     return ALL_KEYS.find(keymap => keymap.description.toUpperCase() === descrption) || null;
}
```